timeseries/src/autogluon/timeseries/models/presets.py

```diff
@@ -193,30 +201,52 @@ def get_preset_models(
             f"hyperparameters must be a dict, a string or None (received {type(hyperparameters)}). "
             f"Please see the documentation for TimeSeriesPredictor.fit"
         )
+    # Handle model names ending with "Model", e.g., "DeepARModel" is mapped to "DeepAR"
+    hyperparameters_clean = {}
+    for key, value in hyperparameters.items():
+        if isinstance(key, str):
+            key = normalize_model_type_name(key)
+        hyperparameters_clean[key] = value
```

Comment:
What if the user provides the following:

```python3
hyperparameters = {
'DeepAR': {'epochs': 5},
'DeepARModel': {'epochs': 10}
}
```

I believe this edge-case is not accounted for in an expected manner atm.

Valid options:
1. Error
2. hyperparameters_clean = {'DeepAR': [{'epochs': 5}, {'epochs': 10}]}
3. Warn and tell user which one takes priority.